pkg/virt-controller/services/net_binding_test.go

```diff
@@ -98,33 +102,39 @@ var _ = Describe("Network Binding", func() {
 			config := &v1.KubeVirtConfiguration{
 				NetworkConfiguration: &v1.NetworkConfiguration{},
 			}
-			_, err := services.NetBindingPluginSidecarList(vmi, config)
+			_, err := services.NetBindingPluginSidecarList(vmi, config, record.NewFakeRecorder(1))
 			Expect(err).To(HaveOccurred())
 		})
 	})
 
-	Context("binding plugin configuration", func() {
-		It("should read registered network binding plugins successfully", func() {
```

Comment:
I don't understand why the original tests were removed. Where are the unit tests checking non sriov plugins or SRIOV as a plugin (and not the old API flow).

Reply:
Sorry, I don't follow, this commit adapt unit tests following the removal of `ReadNetBindingPluginConfiguration` in favor of  `SlirpNetBindingPluginSidecar`